llvm/docs/MIRLangRef.rst

```diff
@@ -807,6 +807,23 @@ For an int eq predicate ``ICMP_EQ``, the syntax is:
 
    %2:gpr(s32) = G_ICMP intpred(eq), %0, %1
 
+Lanemask Operands
```

Comment:
Nit: Please spell it "LaneMask" (capital M) throughout this patch to match the existing codebase.